go/client/cmd_device_remove.go

```diff
@@ -133,8 +159,12 @@ func NewCmdDeviceRemove(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.
 		Usage:        "Remove a device",
 		Flags: []cli.Flag{
 			cli.BoolFlag{
-				Name:  "f, force",
-				Usage: "Override warning about removing the current device.",
+				Name:  "force",
```

Comment:
sorry, seeing the warnings above (and the full commit), makes me think this should be `--force-self`.